xml/obs_best_practice_staging_workflow.xml

```diff
@@ -48,29 +48,29 @@
     different packages all the time. They have to be reviewed, adjusted and
     tested (built) before being accepted. As you can imagine, it becomes
     nonviable to review the packages one by one. Even if the maintainer checks
-    that a package is not broken and merge it, it can break everything else for
+    that a package is not broken and merges it, it can break everything else for
     conflicts with other packages. To deal with these situations, Build Service
     provides what we call Staging Workflow.
   </para>
   <para>
     The idea behind the Staging Workflow is testing the requests incrementally
     by batches. First, a copy of the original project is created, it is called
     Staging Project and is going to act as a playground. The Staging Manager
-    selects some of the requests he consider they belong together and assign the
-    corresponding packages to the Staging Project. This way, the group of
+    selects some of the requests they considers they belong together and assigns the
```

Comment:
Grammar fix: `they` needs to go with `consider`; wording improvement for `belong`:
```suggestion
    selects some of the requests they consider to be belonging together and assigns the
```